README.md

```diff
@@ -22,16 +22,6 @@ Please go to the <a href="https://github.com/YurisCodingClub/html-tag-quiz/discu
 
 [//]: contributor-faces
 
-## How to contribute
-
-This project aims to be a beginner friendly way to your first contribution to open source. If you are looking to make your first contribution, follow the steps below.
-
-Also, read the <a href="https://github.com/YurisCodingClub/accessibility-mentor/blob/main/CONTRIBUTING.md">Contribution Guide</a> before interacting with the repositories.
-
-Images are from the <a href="https://github.com/YurisCodingClub/dev-cv/">Dev CV repository of this organization.</a>
-
-‼️ To get more women and especially people with disabilities into tech, I always assign tickets to them first.
-
 ### CodeNewbies
```

Comment:
Since this is a level three heading (and a few more are coming afterward), they are part of the **How to contribute** section and should therefore also be transferred.